config/betterquesting/DefaultQuests/Quests/SpaceRace-AAAAAAAAAAAAAAAAAAAAIg==/Naphthalene-AAAAAAAAAAAAAAAAAAAKhg==.json

```
@@ -8,7 +8,7 @@
   "properties:10": {
     "betterquesting:10": {
       "autoClaim:1": 0,
-      "desc:8": "To make Naphthalene, combine your Coal Oil Tar with Sulfuric Acid to get Sulfuric Coal Tar Oil.\n\nThen distill it to get Naphthalene. Seem easy? It gets harder.",
+      "desc:8": "To make Naphthalene, combine your Coal Tar Oil with Sulfuric Acid to get Sulfuric Coal Tar Oil.\n\nThen distill it to get Naphthalene. Seems easy? It gets harder.",
```

```suggestion
      "desc:8": "To make Naphthalene, combine your Coal Tar Oil with Sulfuric Acid to get Sulfuric Coal Tar Oil, then distill it to get Naphthalene. Seems easy? It gets harder.",
```

Why the line break in the middle of a sentence?